README.md

```diff
@@ -412,6 +412,92 @@ kustomize build applications/pipeline/upstream/env/cert-manager/platform-agnosti
 ```
 This installs Argo with the runasnonroot emissary executor. Please note that you are still responsible for analyzing the security issues that arise when containers are run with root access and for deciding if the Kubeflow pipeline main containers are run as runasnonroot. It is generally strongly recommended that all user-accessible OCI containers run with Pod Security Standards [restricted](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted).
 
+#### Kubeflow Pipelines - Kubernetes Native API Mode
+
+Kubeflow Pipelines can be deployed in Kubernetes Native API mode, which stores pipeline definitions as Kubernetes Custom Resources instead of using external storage. This mode provides better integration with Kubernetes native tooling and GitOps workflows.
+
+**Install Kubeflow Pipelines in Kubernetes Native API Mode:**
+
+```sh
+# For single-user deployment
+kustomize build applications/pipeline/upstream/env/cert-manager/platform-agnostic-k8s-native | kubectl apply -f -
+
+# For multi-user deployment
+kustomize build applications/pipeline/upstream/env/cert-manager/platform-agnostic-multi-user-k8s-native | kubectl apply -f -
+```
+
+**Key Differences in Kubernetes Native API Mode:**
+
+- Pipeline definitions are stored as `Pipeline` and `PipelineVersion` Custom Resources in Kubernetes
+- No external database required for pipeline storage (metadata still uses external storage)

```

Comment:
either `metadata still stored in external database` or `mlmd still uses external storage` - just `metadata` sounds like the concept of metadata, not the ml-metadata component/dependency

Reply:
Lets ue ml-metadata or whatever the official name is, no unnecessary abbreviations.